### ISSUE TITLE
All the  buttons should have the same sized border

### ISSUE THREAD

**Issue by drapisarda** [homeday-de/homeday-blocks#281]
The full color .btn (--primary --secondary ...) have no border. So if I use a button with a transparent bg and I change the class to color it (homeday homepage header ctas) the button looks like it is moving.

**Comment by ladas-larry** [comment]
Hi @drapisarda, do you have a way how to replicate this issue in storybook?
Class btn has border set to 'none', so far it's got just box-shadow.
Where is the border being assigned?

Is this issue still relevant?

**Comment by drapisarda** [comment]
Most probably is not relevant for this library, because we do not have buttons with a clear border (i.e. white background and blue border). 

We should probably implement one. Then the issue would be that if the style changes on hover, also the button with no clear border, should have a border to keep the sizing and the possible hover effect with no change of sizing.

**Comment by viniciuskneves** [comment]
@drapisarda could you add examples?

**Comment by drapisarda** [comment]
**This was the scenario when I wrote this issue:**
I have a primary button (blue bg, white text). When I hover, I invert the style (white bg, blue text, **blue border**). Said that the idle status button has no border (.btn--primary), at the hover, the button changes size because of the adding of the border. 

**How is it now:**
I noticed now we set `box-sizing: border-box;` now also for the buttons. So this issue is no more relevant.
